tests/test_environment.py

```diff
@@ -35,3 +38,37 @@ def test_environment(environment_path, env):
         traceback.print_exc()
 
     assert no_error_loop
+
+
+@pytest.mark.parametrize('env', ENVS)
+def test_environment2(environment_path, env):
+    """Test battle loop of environments"""
+    no_error_loop = False
+    try:
+        env_path = '.'.join([environment_path, env])
+        env_module = importlib.import_module(env_path)
+        e = env_module.Environment()
+        es = {p: env_module.Environment() for p in e.players()}
+        for _ in range(100):
+            e.reset()
+            for p, e_ in es.items():
+                info = e.diff_info(p)
+                e_.update(info, True)
+            while not e.terminal():
+                actions = {}
+                for player in e.turns():
+                    action = random.choice(es[player].legal_actions(player))
+                    actions[player] = es[player].action2str(action, player)
+                actions = {p: e.str2action(a, p) for p, a in actions.items()}
+                e.steps(actions)
+                for p, e_ in es.items():
+                    info = e.diff_info(p)
+                    e_.update(info, False)
+                e.reward()
+            e.outcome()
```

Comment:
I think this is a bit weird... What do you think?

At least, should the returned value be set something like `_`? (Although this test aims to test not to occur any errors... so the current test could be okay)
```
_ = e.outcome()
```

Reply:
I don't think this current code is strange, but it's ok for me to change these lines as such.

Reply:
Ok, just leave it as it is